working-groups/user-interface/Agenda.md

```diff
@@ -0,0 +1,94 @@
+# Agenda
+
+### Framework for the user-interface (UI) working group meetings (30 - 45 min)
+
+- Welcoming new contributors
+- Quick Update for the past week work + Shoutout to the contributors.
+- Current Work in progress tasks
+- Upcoming Plans for the next week
+- Questions/Blockers
+
+## Agenda for UI working group
+
+***Note-** The agenda speaks about will happen during each meeting, and what will a participant need to do between each meeting i.e the ‘homework’. Week 1 to Week 5 aims to develop a working group(s) notice board.*
+
+### Week 1 - 19th Feb
+
+*Week 1 marks the beginning of the UI working group. During this week we would like to introduce the working group and discuss/study the projects we hope to develop over the next few weeks.*
+
+**Meeting:**
+
+- Introduction to the UI working group
+- Talk about deliverables i.e the TSC working group.
+- Talk about the expected timeline.
+
+**Homework:**
```

Comment:
Is this the right term here?

Reply:
We're referring to this document-- https://larahogan.me/blog/running-working-groups/#plan-your-agenda-arc